doc/source/ScriptingAPI/Context/index.md

```diff
@@ -0,0 +1,34 @@
+# Context #
+
+In Gaffer's parallel computing engine, there is no concept of a single “current time.” All threads evaluate with a Context object, which defines a dictionary of context-related values in which a computation is performed. The most basic entry common to all contexts is the frame number. Other context values include the frame rate, current time in the sequence, and any scene, image, text, and global script context variables.
```

Comment:
I know this is already sounding complicated, but I think it does really help when understanding context variables to have some examples of what scene context variables ( like which location in the scene hierarchy you are evaluating ) and image context variables ( like which tile and channel in the image you are evaluating ) actually are.